src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsCard/DataElementsCard.tsx

```diff
@@ -199,7 +199,17 @@ const DataElementsCard = (props: {
     }
   }, [localSelectedDataElement, codeSystemMap, dataElements]);
   // centralize state one level up so we can conditionally render our child component
-
+  useEffect(() => {
+    setAttributesPresent(false);
+    if (selectedDataElement && selectedDataElement.schema?.eachPath) {
+      selectedDataElement.schema.eachPath((path) => {
```

Comment:
how about using https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some?

Reply:
`eachPath` is from models. I did not realize that. Please ignore my comment. my bad.